docs/layouts/_default/baseof.html

```diff
@@ -0,0 +1,93 @@
+<!DOCTYPE html>
+<html lang="{{ .Site.Language.Lang }}" dir="{{ .Site.Language.LanguageDirection | default "ltr" }}">
+<head>
+  {{ partial "docs/html-head" . }}
+  {{ partial "docs/inject/head" . }}
+</head>
+<body dir="{{ .Site.Language.LanguageDirection | default "ltr" }}">
+  <input type="checkbox" class="hidden toggle" id="menu-control" />
+  <input type="checkbox" class="hidden toggle" id="toc-control" />
+  <main class="container flex">
+    <aside class="book-menu">
+      <div class="book-menu-content">
+        {{ template "menu" . }} <!-- Left menu Content -->
+      </div>
+    </aside>
+
+    <div class="book-page">
+      <div class="book-search">
```

Comment:
This file is a complete copy out of the `themes` directory so we can just use [hugo's theme customization pattern](https://bwaycer.github.io/hugo_tutorial.hugo/themes/customizing/) instead of changing the theme directly. The only edit is this `div` and it's contents.